eng/common/scripts/Create-APIReview.ps1

```diff
@@ -0,0 +1,104 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,
+  [Parameter(Mandatory=$True)]
+  [string] $APILabel,
+  [string] $PackageName = ""
+)
+
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel)
+{
+    $multipartContent = [System.Net.Http.MultipartFormDataContent]::new()
+    $FileStream = [System.IO.FileStream]::new($filePath, [System.IO.FileMode]::Open)
+    $fileHeader = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("form-data")
+    $fileHeader.Name = "file"
+    $fileHeader.FileName = $packagename
+    $fileContent = [System.Net.Http.StreamContent]::new($FileStream)
+    $fileContent.Headers.ContentDisposition = $fileHeader
+    $fileContent.Headers.ContentType = [System.Net.Http.Headers.MediaTypeHeaderValue]::Parse("application/octet-stream")
+    $multipartContent.Add($fileContent)
+
+
+    $stringHeader = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("form-data")
+    $stringHeader.Name = "label"
+    $StringContent = [System.Net.Http.StringContent]::new($apiLabel)
+    $StringContent.Headers.ContentDisposition = $stringHeader
+    $multipartContent.Add($stringContent)
+
+    $headers = @{
+        "ApiKey" = $apiKey;
+        "content-type" = "multipart/form-data"
+    }
+
+    try
+    {
+        $Response = Invoke-WebRequest -Method 'POST' -Uri $uri -Body $multipartContent -Headers $headers
+        $StatusCode = $Response.StatusCode
+    }
+    catch
+    {
+        $StatusCode = $_.Exception.Response.StatusCode
+    }
+
+    return $StatusCode
+}
+
+
+. (Join-Path $PSScriptRoot common.ps1)
+$packages = @{}
+
+if (Test-Path "Function:Find-Artifacts-For-Apireview")
+{
+    $packages = Find-Artifacts-For-Apireview($ArtifactPath, $PackageName)
```

Comment:
```suggestion
    $packages = Find-Artifacts-For-Apireview $ArtifactPath $PackageName
```
You cannot call this as a standard function in PS the are more like command line arguments. Have you ran this code yet? If not I'd like to wait until we can test it in the repo's before we merge this into eng\common.

Reply:
This did work actually and my test run had created below API review. https://apiviewstaging.azurewebsites.net/Assemblies/Review/609cf46560804ae6858e9aab786e73e5

I will anyway change to code to PS standard format as per suggestion above.

Reply:
Created a test review for Java using latest changes: https://apiviewstaging.azurewebsites.net/Assemblies/Review/d2cfa2fb40de477491ee614f826e83fc